prov/psm/src/psmx_init.c

```diff
@@ -398,7 +399,7 @@ PSM_INI
 	int check_version;
 	int err;
 
-	fi_log_init();
+	psm_handle = fi_log_register_provider("psm");
```

Comment:
Sean would you see extending the fi_provider struct to include a field that serves as an index into the current bitmap idea?   I'm not seeing how the fi_prov struct could help much here, since the providers don't know about the fi_prov struct.  It sounds cludgy, but the upper layer could set a "debug" handle in the providers' returned fi_provider struct as part of the fi_register_provider.